VideoLocker/src/main/java/org/edx/mobile/util/AppStoreUtils.java

```diff
@@ -0,0 +1,60 @@
+package org.edx.mobile.util;
+
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.View;
+import android.widget.Toast;
+
+import org.edx.mobile.BuildConfig;
+import org.edx.mobile.R;
+
+/**
+ * Utility class for interacting with an app store, or the
+ * Play Store specifically.
+ */
+public final class AppStoreUtils {
+    // Make this class non-instantiable
+    private AppStoreUtils() {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Open an app store to display the app.
+     *
+     * @param context A Context for starting the new Activity.
+     */
+    public static void openAppInAppStore(final Context context) {
+        Intent intent = new Intent(Intent.ACTION_VIEW);
+        intent.addCategory(Intent.CATEGORY_BROWSABLE);
+        intent.setData(Uri.parse("market://details?id=" + BuildConfig.APPLICATION_ID));
+        try {
+            context.startActivity(intent);
+        } catch (ActivityNotFoundException e) {
+            intent.setData(Uri.parse("https://play.google.com/store/apps/details?id=" +
+                            BuildConfig.APPLICATION_ID));
```

Comment:
> Let's not solve for apps intercepting the `market://` URI, but lets definitely make both of these configurable.

Done in commit 3e85fbcf6f164567e8885a433677caca9bd72503. The name and structure of the flags still needs to be finalized in the wiki (and then added in the existing configs), but I have tentatively assumed a `APP_UPDATE_URIS` top-level object containing `NATIVE` and `WEB` fields.

> And perhaps, also have a method `boolean canOpenAppInAppStore()` that returns true if either of the URIs are configured and resolvable. And then only show the "Update" link if it returns true. Make sense?

Done in commit 3864c8c5da4bdf984de9b495d2559179ca33d21b.